### PR TITLE
Clarify sort order when specified limit

### PR DESCRIPTION
**Pull request by AsaiToshiya**
From reading #579, #663 and the implementation of major relays, it seems that the descending order of the `created_at` is a common understanding, so this PR clarified it.

### CODE REVIEW

**Comment by Egge7** [comment]
Just to make it really clear I would rephrase it further:

"When `limit: n` is present it is assumed that the events returned in the initial query will be the last `n` events ordered by `created_at`"

**Comment by AsaiToshiya** [comment]
Thanks for your feedback.
